include/diaspora.php

```diff
@@ -1682,7 +1717,7 @@ private function receive_message($importer, $data) {
 	 *
 	 * @return bool always true
 	 */
-	private function receive_participation($importer, $data) {
+	private static function receive_participation($importer, $data) {
```

Comment:
Naive question: Shouldn't this becimplemented if we expect event participations from Diaspora?